packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -356,38 +356,20 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
         if (!snapshotTree) {
             return null;
         }
-        // Decode commit paths
-        const commits = {};
 
-        for (const key of Object.keys(snapshotTree.commits)) {
-            commits[decodeURIComponent(key)] = snapshotTree.commits[key];
-        }
-
-        let finalTree: api.ISnapshotTree;
-        // For container loaded from detach new summary, we will not have a commit for ".app" in downloaded summary as the client uploaded both
-        // ".app" and ".protocol" trees by itself. For other summaries, we will have ".app" as commit because client previously only uploaded the
-        // app summary.
-        if (commits && commits[".app"]) {
```

Comment:
Can you please run it by Jatin and maybe Marcelo?
SPO for some time used commits. I think it's gone, but I'm not 100% sure.
If this flow is still needed, we can define SPO specific version of ISnapshotTree - it should be local only in this file as outside of the SPO driver we expose proper ISnapshotTree without commits (i.e. this code was flattening it in the past)

Reply:
@jatgarg - please take a look

Reply:
@marcmasmsft @znewton  can you confirm whether we still need the commit on the SPO ? We don't think so but want to confirm. Thanks!

Reply:
Marcelo just confirmed they do not use commit anymore.

Reply:
Yeah we don't use it anymore. We can remove it.